contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -177,29 +176,35 @@ PRAGMA_WARNING_DISABLE_VS(4355)
     // create a random uuid, we don't need crypto strength here
     const boost::uuids::uuid random_uuid = boost::uuids::random_generator()();
 
-    context = t_connection_context{};
+    get_context() = t_connection_context{};
```

Comment:
Seems you could reduce delta noise in this function and also help the compiler with:
```
auto context = get_context() = t_connection_context{};
```

Reply:
This hides a member variable from a derived class, but it does limit the changes. Done.